patterns/hello_world_pubsub.py

```diff
@@ -106,12 +102,15 @@ def on_failed_publish(self, e: "FailedPublishEvent"):
         print("Connected and Subscribed! Ready to publish\n")
     try:
         while not SHUTDOWN:
+            # Creating a dynamic outbond message 
+            outbound_msg = messaging_service.message_builder() \
+                .with_application_message_id(f'sample_id {msgSeqNum}') \
+                .with_property("application", "samples") \
+                .with_property("language", "Python") \
+                .build(f'{message_body} --> {msgSeqNum}')
             # Direct publish the message
             direct_publisher.publish(destination=Topic.of(TOPIC_PREFIX + f"/python/{unique_name}/{msgSeqNum}"), message=outbound_msg)
             msgSeqNum += 1
-            # Modifying the outbond message instead of creating a new one
-            outbound_msg.solace_message.message_set_binary_attachment_string(f'{message_body} --> {msgSeqNum}')
-            outbound_msg.solace_message.set_message_application_message_id(f'sample_id {msgSeqNum}')
             time.sleep(0.1)
```

Comment:
time.sleep should not be necessary when using back the default back pressure elastic. Why is this here?

Reply:
I had it for demo purposes to show better show the flow of messages. I have however commented it out but kept it in case